### ISSUE TITLE
Within a challenge, always open links in a new tab

### ISSUE THREAD

**Issue by rcanzanese** [CTFd/CTFd#1150]
My participants were always asking for me to make links within the challenges open in a new tab by default, so that they didn't keep accidentally navigated away from the CTF window of the challenge they were working on. 

This is a simple change that I already have made.  I can submit a PR if there is enough interest.

**Comment by ColdHeat** [comment]
This is a duplicate of https://github.com/CTFd/CTFd/issues/875 which was already implemented in 2.2.0 but I'm curious how you implemented this.

**Comment by rcanzanese** [comment]
@ColdHeat  awesome.  I did it in the `challege.postRender` function.

```
--- a/CTFd/plugins/challenges/assets/view.js
+++ b/CTFd/plugins/challenges/assets/view.js
@@ -15,7 +15,13 @@ window.challenge.render = function (markdown) {
 
 
 window.challenge.postRender = function () {
-
+    var el = document.getElementsByClassName('challenge-desc')[0];
+    var anchors = el.getElementsByTagName('a');
+    for (var i=0; i<anchors.length; i++){
+        if (anchors[i].hostname != window.location.hostname) {
+            anchors[i].setAttribute('target', '_blank');
+        }
+    }
 };
```
